Cargo.toml

```diff
@@ -45,7 +45,7 @@ rustls-tls = [
 
 [dependencies]
 anyhow = "1.0"
-tokio = { version = "1.0", features = ["macros", "rt-multi-thread", "time"] }
+tokio = { version = "1.0", features = ["macros", "rt-multi-thread", "time", "process"] }
```

Comment:
The new features and dependencies added here are for things in the unit tests and associated binaries (like podsmiter)

Reply:
Is this something that can be set in `dev-dependencies`?

Reply:
Nope. There isn't a way to turn on the feature conditionally. I don't think you can have a duplicate dep in `dev-dependencies` can you?

Reply:
I'm pretty sure `kube` does this for `k8s-openapi` https://github.com/clux/kube-rs/blob/a6cc8995183a9ff9b0d7deb2ebad11dfa4ca31bb/kube/Cargo.toml#L86

Reply:
Fixed!